test/io/perun_test.clj

```diff
@@ -30,15 +32,15 @@
    m msg  MSG  str "message shown on failure"]
   (boot/with-pass-thru fileset
     (let [file (boot/tmp-get fileset path)]
-      (is (contains? (pm/+meta-key+ file) key) msg))))
+      (is (contains? (pm/meta-from-file fileset file) key) msg))))
 
 (deftask value-test
   [p path     PATH    str "path of the file to test"
    v value-fn VALUEFN edn "the value to test (optional)"
    m msg      MSG     str "message shown on failure"]
   (boot/with-pass-thru fileset
     (let [file (boot/tmp-get fileset path)]
-      (is (and (not (nil? file)) (value-fn file)) msg))))
+      (is (and (not (nil? file)) (value-fn fileset file)) msg))))
 
 (deftask content-test
```

Comment:
A test exercises some functionality and then checks that it's behavior is correct.  A check or assert only makes sure that some invariant is true.